webapp/i18n/en.json

```diff
@@ -6,21 +6,25 @@
   "47FYwb": "Cancel",
   "7C5R1Z": "Real-time communication daemon is a service built to offload calls onto your own WebRTC services and efficiently support scalable and secure deployments. <featureLink>Learn more about this feature</featureLink>.",
   "9tBhzB": "Upgrade now",
+  "CkbjCZ": "Enable recordings for calls",
   "EaucLA": "View plans",
-  "H79wnj": "Welcome to your Mattermost Enterprise trial! It expires on {trialExpirationDate}. You now have access to <rtcdDocsLink>RTCD services</rtcdDocsLink>, <guestAccountsLink>guest accounts</guestAccountsLink>, <autoComplianceReportsLink>automated compliance reports</autoComplianceReportsLink>, and <mobileSecureNotificationsLink>mobile secure-ID push notifications</mobileSecureNotificationsLink>, among many other features. View all features in our <documentationLink>documentation</documentationLink>.",
   "Igo1xs": "It looks like something went wrong with your Enterprise trial request. You can try again later or <supportLink>contact Support</supportLink> if the error persists.",
   "KaiRbV": "Calls are a quick, audio-first, way to interact with your team. Get the full calls experience when you start a free, 30-day trial.",
   "M6lXfS": "Set up RTCD services",
   "N2IrpM": "Confirm",
   "TdTXXf": "Learn more",
+  "ToKZOz": "<featureLink>Learn more about this feature</featureLink>.",
```

Comment:
@streamer45, will this link to the docs and, if so, do you have a link to the spec so I can update the docs.

Reply:
@justinegeffen I don't yet but I am planning to work on some documentation for this feature as soon as we are done with the main implementation. Will ping you as soon as I have something to review.